tests/platform_tests/api/test_psu_fans.py

```diff
@@ -187,12 +187,18 @@ def test_is_replaceable(self, platform_api_conn):
     #
 
     def test_get_speed(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost, platform_api_conn):
-
+        duthost = duthosts[enum_rand_one_per_hwsku_hostname]
         for j in range(self.num_psus):
             num_fans = psu.get_num_fans(platform_api_conn, j)
+            fans_skipped = 0
```

Comment:
Is this new variable "fans_skipped" really needed ?

Reply:
removing too